File: SourceCode/C/day047.c
/* Author: Ahmed A. M. ALOMAR
Date: February 16 - 2020
Description:
This program is an example of the Continue statement in a loop  */


#include <stdio.h>

int main()
{
  int x = 0;
  while (x < 10)
  {
    printf("x = %d \n", x);
    x++;
    if (x == 5)
    {
        printf("Continue Statement %d \n", x);
        printf("This is temporary \n");
        continue;
    }
  }

  return 0;
}

File: SourceCode/Python/day047.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: May 27 - 2020
#
# Description:
# * Example of Class Protected Methods
# ------------------------------------ #


class Car():
  def __init__(self):
# Private attributes
    self._make = "FORD"
    self._year = 2020

# Using a protected method
  def get_production_year(self):
    return self._year

daily_car = Car()
print(daily_car.get_production_year())
